verticapy/_utils/_sql/_format.py

```diff
@@ -1663,6 +1663,11 @@ def schema_relation(relation: Any, do_quote: bool = True) -> tuple[str, str]:
             )
             if rel_transf.split('"')[2] != ".":
                 raise ParsingError("The format of the input relation is incorrect.")
+        elif quote_nb == 0 and dot_nb == 2:
```

Comment:
Can you please add a unit test for this case?

Reply:
We previously did not have any unit tests. I have added unit tests for all cases.